app/services/docker/pluginService.js

```diff
@@ -20,7 +20,7 @@ angular.module('portainer.services')
     return deferred.promise;
   };
 
-  servicePlugins = function(systemOnly, pluginType, pluginVersion) {
+  var servicePlugins = function(systemOnly, pluginType, pluginVersion) {
```

Comment:
Actually, if that function is not exposed via the service you can just use `function servicePlugins(systemOnly, pluginType, pluginVersion)` instead of assigning it to a variable.